content/patterns/button/button-pattern.html

```diff
@@ -14,92 +14,93 @@
+            しかし、より良い解決策は、機能と ARIA の役割が一致するように視覚デザインを調整することです。
           </p>
         </div>
       </section>
 
       <section id="examples">
-        <h2>Examples</h2>
+        <h2>例</h2>
         <ul>
-          <li><a href="examples/button.html">Button Examples</a>: Examples of clickable HTML <code>div</code> and <code>span</code> elements made into accessible command and toggle buttons.</li>
+          <li><a href="examples/button.html">ボタンの例</a> ：クリック可能なHTMLの <code>div</code> 要素及び <code>span</code> 要素を使って、アクセシブルなコマンドボタン及びトグルボタンを作った例。</li>
           <li>
-            <a href="examples/button_idl.html">Button Examples (IDL)</a>: Examples of clickable HTML <code>div</code> and <code>span</code> elements made into accessible command and toggle buttons.
-            This example uses the <a class="specref" href="#idl-interface">IDL Interface</a>.
+            <a href="examples/button_idl.html">ボタンの例（IDL）</a> ：クリック可能なHTMLの <code>div</code> 要素及び <code>span</code> 要素を使って、アクセシブルなコマンド及びトグルボタンを作った例。
+            この例では <a class="specref" href="#idl-interface">IDLインターフェイス</a> を用いています。
           </li>
         </ul>
       </section>
 
       <section id="keyboard_interaction">
-        <h2>Keyboard Interaction</h2>
-        <p>When the button has focus:</p>
+        <h2>キーボード操作</h2>
+        <p>ボタンにフォーカスがある場合：</p>
         <ul>
-          <li><kbd>Space</kbd>: Activates the button.</li>
-          <li><kbd>Enter</kbd>: Activates the button.</li>
+          <li><kbd>Space</kbd> ：ボタンをアクティブにします。</li>
+          <li><kbd>Enter</kbd> ：ボタンをアクティブにします。</li>
           <li>
-            Following button activation, focus is set depending on the type of action the button performs. For example:
+            ボタンをアクティブにした後、フォーカスの設定はボタンが実行するアクションの種類によって異なります。例えば：
             <ul>
               <li>
-                If activating the button opens a dialog, the focus moves inside the dialog.
-                (see <a href="../dialog-modal/dialog-modal-pattern.html">dialog pattern</a>)
+                ボタンを押すとダイアログが開く場合、フォーカスはダイアログ内に移動します。
+                （ <a href="../dialog-modal/dialog-modal-pattern.html">ダイアログ パターン</a> を参照）
               </li>
               <li>
-                If activating the button closes a dialog, focus typically returns to the button that opened the dialog unless the function performed in the dialog context logically leads to a different element.
-                For example, activating a cancel button in a dialog returns focus to the button that opened the dialog.
-                However, if the dialog were confirming the action of deleting the page from which it was opened, the focus would logically move to a new context.
+                ボタンを押すとダイアログが閉じる場合、ダイアログのコンテキストで実行された機能が論理的に別の要素につながらない限り、フォーカスは通常、ダイアログを開いたボタンに戻ります。
+                例えば、ダイアログのキャンセルボタンを押すと、フォーカスはダイアログを開いたボタンに戻ります。
+                しかし、ダイアログが開かれたページを削除する操作を確認するものであった場合、フォーカスは論理的に新しいコンテキストに移動します。
```

Comment:
ダイアログが開かれたページそのものであることが「the」に込められてる感があったので、日本語ではもう少し明示的に訳してもよいかもしれません。

```suggestion
                しかし、ダイアログがそのダイアログが開かれたページを削除する操作を確認するものであった場合、フォーカスは論理的に新しいコンテキストに移動します。
```